SPECS/python-werkzeug/python-werkzeug.spec

```diff
@@ -33,22 +48,34 @@ Werkzeug started as simple collection of various utilities for WSGI applications
 %prep
 %autosetup -n werkzeug-%{version} -p1
 
+%generate_buildrequires
+%pyproject_buildrequires %{?with_tests:-t}
+
 %build
-%py3_build
+%pyproject_wheel
 
 %install
-%py3_install
+%pyproject_install
+%pyproject_save_files werkzeug
 
 %check
-pip3 install pytest hypothesis
-LANG=en_US.UTF-8 PYTHONPATH=./  python3 setup.py test
+pip3 install tox==4.6.3 tox-current-env
```

Comment:
Btw. why do we need to pin this to an older version? I'm not saying it's wrong, but let's leave a comment.